demo/config/panda_config.yaml

```diff
@@ -0,0 +1,44 @@
+# planning group names
+group_name: "panda_arm"
```

Comment:
Let's use more descriptive parameter names, i.e. "arm_group_name", "end_effector_name", "gripper_group_name", "ik_frame".

Reply:
The group names, like panda_arm and hand are part of the default panda_moveit_config. This is why they are so uninspiredly named. For the default parameter names given from the pnh.parama<>() I will give better descriptions.